docs/user_documentation/import-data/guide-quick-upload.md

```diff
@@ -23,17 +21,32 @@ MOLGENIS supports multiple data types like booleans, date fields, numbers etc.
 5. To get __BOOLEAN__, submit strings in the format _TRUE_ or _FALSE_
 6. Other values are inserted into the database as __STRING__. If a piece of text is longer then 255 characters, we use __TEXT__
 
-## Data placement
+## Data structure
+MOLGENIS has an explicit data structure within the application. You need to know the basic terminology to understand how the data is structured.
 
-Data is imported into the MOLGENIS database as a single _data table_
-_Data tables_ are grouped within packages
+### Terminology
```

Comment:
Following our offline discussion: your entity types are stored in a package that you are allowed to write content into (usually the package of the group that the user is a member of).